### DISCUSSION TITLE
Oct Newsletter - Cloudberry Database(2023/10)

### DISCUSSION THREAD

**Discussion by tuhaihe** [apache/cloudberry#274] [Announcements]
Hey there! Welcome to our October newsletter for the Cloudberry Database project! 

We're so excited to have you on board with us. In this newsletter, we'll be sharing all the latest updates, improvements, and contributions made by our amazing community. From cool new features to pesky bug fixes, we have a lot to chat about, so let's get started!

## Pull Requests

- #272 by @Zhangbaowen-Hashdata
- #271 by @lss602726449
- #268 by @tuhaihe
- #267 by @TomShawn
- #266 by @wenchaozhang-123
- #265 by @avamingli
- #263 by @avamingli
- #261 by @Ray-Eldath
- #260 by @avamingli
- #257 by @avamingli
- #256 by @smartyhero
- #255 by @HuSen8891
- #254 by @avamingli
- #252 by @wenchaozhang-123
- #249 by @roseduan
- #248 by @avamingli
- #247 by @avamingli
- #242 by @MisterRaindrop
- #240 by @Zhangbaowen-Hashdata
- #238 by @avamingli
- #237 by @tuhaihe
- #236 by @HuSen8891
- #235 by @Zhangbaowen-Hashdata
- #234 by @wenchaozhang-123
- #232 by @wenchaozhang-123

## Proposals & Ideas

- #227 by @RyanWei
- #243 by @avamingli

## Issues

- #273 by @congxuebin
- #270 by @congxuebin
- #262 by @avamingli
- #259 by @avamingli
- #253 by @congxuebin
- #251 by @congxuebin
- #250 by @smartyhero
- #246 by @smartyhero
- #245 by @smartyhero
- #244 by @smartyhero

## Contributors

🎈️🎊️ Thanks to the following contributors for helping make Cloudberry Database better this month:

> @Zhangbaowen-Hashdata, @lss602726449, @tuhaihe, @TomShawn, @wenchaozhang-123, @avamingli, @Ray-Eldath, @smartyhero, @HuSen8891, @roseduan, @MisterRaindrop, @congxuebin, @RyanWei

## Join us

The Cloudberry Database community welcomes everyone to contribute, regardless of their level of experience. We encourage all types of contributions, no matter how small. Our [contribution guide](https://cloudberrydb.org/contribute/how-to-contribute) is available to help you get started with the process.

In addition, we offer various channels for community members to discuss, seek help, provide feedback, and chat. You can find support [here](https://cloudberrydb.org/support).

Join us and be part of our community!

## Past Newsletter

- #225 
- #181 

-------

[![Slack](https://img.shields.io/badge/Join_Slack-6a32c9)](https://communityinviter.com/apps/cloudberrydb/welcome) [![Twitter Follow](https://img.shields.io/twitter/follow/cloudberrydb)](https://twitter.com/cloudberrydb) [![Website](https://img.shields.io/badge/Visit%20Website-eebc46)](https://cloudberrydb.org)

**Comment by TomShawn** [comment]
Great!~
